### PR TITLE
Add docs for image runner resourceProfile

### PR DESCRIPTION
**Pull request by tianfeng92**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/orchestrate/saucectl-configuration.md [hunk 1]
```diff
@@ -184,6 +184,20 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
+### `resourceProfile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. The default value is set to c1m1.
+
+```yaml
+resourceProfile: c2m2
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:198]: Just for more context. We should slowly add more parent context as we edit/expand the docs. Lowers the likelihood that the user makes a mistake:
```suggestion
suites:
 - name: "saucy test"
    resourceProfile: c2m2
```

File: docs/orchestrate/saucectl-configuration.md [hunk 2]
```diff
@@ -184,6 +184,20 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
 ```
+---
+
+### `resourceProfile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. The default value is set to c1m1.
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:195]: `c9m9` would not be a valid value though, right? We should list the available options out.

Comment (tianfeng92) [reply docs/orchestrate/saucectl-configuration.md:195]: Oh sure!

File: docs/orchestrate/saucectl-configuration.md [hunk 3]
```diff
@@ -184,6 +184,22 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. Avaiable options are: `c1m1`, `c2m2` and `c3m3`. The default value is set to `c1m1`.
+
+```yaml
+suites:
+ - name: "saucy test"
+    resourceProfile: c2m2
```
Comment (FriggaHel) [inline docs/orchestrate/saucectl-configuration.md:200]: Shouldn't this be that ?

```suggestion
  resourceProfile: c2m2
```

File: docs/orchestrate/saucectl-configuration.md [hunk 4]
```diff
@@ -184,6 +184,22 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. Avaiable options are: `c1m1`, `c2m2` and `c3m3`. The default value is set to `c1m1`.
+
+```yaml
+suites:
+ - name: "saucy test"
+  resourceProfile: c2m2
```
Comment (mhan83) [inline docs/orchestrate/saucectl-configuration.md:200]: Spacing still looks off?

File: docs/orchestrate/saucectl-configuration.md [hunk 5]
```diff
@@ -184,6 +184,22 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
 ```
+---
+
+### `resourceProfile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. Avaiable options are: `c1m1`, `c2m2` and `c3m3`. The default value is set to `c1m1`.
```
Comment (aliciakrebs) [inline docs/orchestrate/saucectl-configuration.md:195]:
```suggestion
The larger the level value, the higher the allocated CPU and memory resources. Available options are: `c1m1`, `c2m2`,  and `c3m3`. The default value is set to `c1m1`.
```

File: docs/orchestrate/saucectl-configuration.md [hunk 6]
```diff
@@ -184,6 +184,22 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
 ```yaml
 workload: webdriver
 ```
+---
+
+### `resourceProfile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+This field sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
```
Comment (aliciakrebs) [inline docs/orchestrate/saucectl-configuration.md:193]:
```suggestion
Sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/orchestrate/saucectl-configuration.md (hunk 4 above)
Comment (tianfeng92) [reply docs/orchestrate/saucectl-configuration.md:200]: 🤔 something interesting. https://github.com/saucelabs/sauce-docs/pull/2009#pullrequestreview-1434199543

Let me check it

Comment (tianfeng92) [reply docs/orchestrate/saucectl-configuration.md]: Fixed!

File: docs/orchestrate/saucectl-configuration.md [hunk 7]
```diff
@@ -184,6 +184,22 @@ Tell Sauce Orchestrate what kind of entrypoint process you are running. `workloa
...
 ```
+---
+
+### `resourceProfile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Sets the CPU and memory limits for the container. The format for specifying the limits is as follows: `<CPU><level><memory><level>`.
+
+The larger the level value, the higher the allocated CPU and memory resources. Available options are: `c1m1`, `c2m2`,  and `c3m3`. The default value is set to `c1m1`.
```
[the remark this answers is not in the document]
Comment (mhan83) [reply docs/orchestrate/saucectl-configuration.md:195]: The doggo is technically not wrong, there is an extra space there. Good doggo.
```suggestion
The larger the level value, the higher the allocated CPU and memory resources. Available options are: `c1m1`, `c2m2`, and `c3m3`. The default value is set to `c1m1`.
```

**Review by mhan83 — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

**Pull request merged**